pkg/plugins/plugins.go

```diff
@@ -217,7 +217,7 @@ func (scanner *PluginScanner) loadPluginJson(pluginJsonFilePath string) error {
 	loader = reflect.New(reflect.TypeOf(pluginGoType)).Interface().(PluginLoader)
 
 	// External plugins need a module.js file for SystemJS to load
-	if !strings.HasPrefix(pluginJsonFilePath, setting.StaticRootPath) {
+	if !strings.HasPrefix(pluginJsonFilePath, setting.StaticRootPath) && pluginCommon.Type != "renderer" {
```

Comment:
can we extract this to && !isBackendOnly(pluginCommon.Type)

function isBackendOnly(string type) ... 

Should be clearer and we will very likely add more plugins that only have backend parts

Reply:
Good suggestion, I will take a look again.